proto/endorsement.proto

```diff
@@ -45,6 +45,7 @@ message VMGoldenMeasurement {
   bytes ca_bundle = 6;
   VMSevSnp sev_snp = 7;
   VMTdx tdx = 8;
+  SvsmMeasurement svsm = 9;
```

Comment:
This measurement is still technology-specific, so I'd expect this to be in VMSevSnp.